app/views/widgets/single_column/_messaging_2_column_youtube_video.html

```diff
@@ -17,7 +17,18 @@ <h1 class="title">Sample Message With Photo</h1>
             Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sit amet libero ultrices, aliquam ligula eu, sodales ligula. Ut maximus mauris tincidunt maximus hendrerit. Nullam velit felis, ultricies sit amet efficitur at, aliquet vitae velit. Phasellus dui magna, condimentum id fringilla sit amet, ultrices a magna. Ut semper, leo nec efficitur facilisis, nisi mi faucibus odio, ac ornare orci urna hendrerit
           </p>
         </div>
-        <div class="actions"><a class="theme-button">This is a button</a></div>
+        <div class="actions">
```

Comment:
Is there a reason for only adding the dropdown to 2 of the sample partials? (`app/views/widgets/single_column/_messaging_2_column_youtube_video` and `app/views/widgets/single_column/_messaging_2_column_vimeo_video`) It should also be added to the partials `messaging_2_column`, `messaging_2_column_text_left`, `messaging_2_column_video_dark_bg`, `messaging_2_column_with_bg_image`. Since they can be used in those areas on the live site, we should have sample widgets in cascade assets that show them.